Description: Update man page references
 Upstream ships with a lot of wrapper scripts that are not used by Debian,
 as we have more convenient ways of bringing up a PPPoE interface. Note that
 pppoe.conf is a config file used by the wrapper scripts.
Author: David Kimdon <dwhedon@debian.org>
Bug-Debian: https://bugs.debian.org/71840
Forwarded: no
Reviewed-By: Andreas Barth <aba@not.so.argh.org>
Last-Update: 2016-06-21

Index: rp-pppoe/man/pppoe-relay.8
===================================================================
--- rp-pppoe.orig/man/pppoe-relay.8
+++ rp-pppoe/man/pppoe-relay.8
@@ -119,6 +119,4 @@ clients on the eth1, eth2 and eth3 netwo
 The \fBpppoe\fR home page is \fIhttp://www.roaringpenguin.com/pppoe/\fR.
 
 .SH SEE ALSO
-pppoe-start(8), pppoe-stop(8), pppoe-connect(8), pppd(8), pppoe.conf(5),
-pppoe(8), pppoe-setup(8), pppoe-status(8), pppoe-sniff(8), pppoe-server(8)
-
+pppd(8), pppoe(8), pppoe-sniff(8), pppoe-relay(8), /usr/share/doc/pppoe/README.Debian.gz
Index: rp-pppoe/man/pppoe-server.8
===================================================================
--- rp-pppoe.orig/man/pppoe-server.8
+++ rp-pppoe/man/pppoe-server.8
@@ -203,6 +203,4 @@ It is \fInot\fR a high-performance serve
 The \fBpppoe\fR home page is \fIhttp://www.roaringpenguin.com/pppoe/\fR.
 
 .SH SEE ALSO
-pppoe-start(8), pppoe-stop(8), pppoe-connect(8), pppd(8), pppoe.conf(5),
-pppoe(8), pppoe-setup(8), pppoe-status(8), pppoe-sniff(8), pppoe-relay(8)
-
+pppd(8), pppoe(8), pppoe-sniff(8), pppoe-relay(8), /usr/share/doc/pppoe/README.Debian.gz
Index: rp-pppoe/man/pppoe-sniff.8
===================================================================
--- rp-pppoe.orig/man/pppoe-sniff.8
+++ rp-pppoe/man/pppoe-sniff.8
@@ -72,6 +72,4 @@ exit.
 The \fBpppoe\fR home page is \fIhttp://www.roaringpenguin.com/pppoe/\fR.
 
 .SH SEE ALSO
-pppoe-start(8), pppoe-stop(8), pppoe-connect(8), pppd(8), pppoe.conf(5),
-pppoe(8), pppoe-setup(8), pppoe-status(8), pppoe-server(8), pppoe-relay(8)
-
+pppd(8), pppoe(8), pppoe-sniff(8), pppoe-relay(8), /usr/share/doc/pppoe/README.Debian.gz
Index: rp-pppoe/man/pppoe.8
===================================================================
--- rp-pppoe.orig/man/pppoe.8
+++ rp-pppoe/man/pppoe.8
@@ -252,5 +252,4 @@ with much inspiration from an earlier ve
 The \fBpppoe\fR home page is \fIhttp://www.roaringpenguin.com/pppoe/\fR.
 
 .SH SEE ALSO
-pppoe-start(8), pppoe-stop(8), pppoe-connect(8), pppd(8), pppoe.conf(5), pppoe-setup(8), pppoe-status(8), pppoe-sniff(8), pppoe-server(8), pppoe-relay(8)
-
+pppd(8), pppoe-sniff(8), pppoe-server(8), pppoe-relay(8), /usr/share/doc/pppoe/README.Debian.gz
